src/framework/Elsa.Studio.Shared/Components/DataPanel.razor

```diff
@@ -21,11 +24,18 @@
                 <td>
                     @if (!string.IsNullOrWhiteSpace(item.Value.Link))
                     {
-                        <MudLink Href="@item.Value.Link">@item.Value.Text</MudLink>
-                    }
-                    else
-                    {
-                        <span>@item.Value.Text</span>
+                            <MudLink Href="@item.Value.Link">@item.Value.Text</MudLink>
+                        }
+                        else
+                        {
+                            @if (@item.Key == "Created" || @item.Key == "Updated" || item.Key == "Finished")
```

Comment:
Although I like that we would have better display support for Timestamp values, I don't like that this DataPanel component has specific logic for these properties. Either we should revert this change, or, even better, we make DataPanel support components as values in addition to string values.

Reply:
Will fix later